app/api/routes/order_status.py

```diff
@@ -0,0 +1,48 @@
+from uuid import UUID
+from app.api.exceptions.global_exceptions import InternalServerErrorException, StatusAlreadyExistsException, StatusInUseException, StatusNameInvalidException, StatusNotFoundException
+from app.api.services.order_status_service import OrderStatusService
+from fastapi import APIRouter, HTTPException, status
+
+router = APIRouter()
+order_status_service = OrderStatusService
+
+@router.post("/statuses/", status_code=status.HTTP_201_CREATED)
+def create_order_status(name: str):
+    if not name:
```

Comment:
No need for this check, you're defining it as a required parameter `name: str`, not `name: str | None = None`.